packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -195,13 +73,31 @@ export default function EntitiesSavedStates( { isOpen, close } ) {
 		close( entitiesToSave );
 	};
 
+	const [ isReviewing, setIsReviewing ] = useState( false );
+	const toggleIsReviewing = useCallback(
+		() => setIsReviewing( ! isReviewing ),
```

Comment:
You can use the functional updater and avoid recreating the callback.

```js
setState( value => ! value )
```

Reply:
Updated.  I think this is what you were referring to?